dashboard/app/views/devise/mailer/confirmation_instructions.html.erb

```
@@ -2,8 +2,6 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, :confirmation_token => @token) %></p>
-
```

should we remove this view altogether? I'm assuming it's no longer used at all, but if it is unintentionally referenced somewhere, i think we'd prefer to report an error than send a user an email missing a link.

---

This is attempting to do the minimal amount of work to remove `:confirmable` from the User model. This view, as well as other stuff, is being removed in #14512.